02-oop-inheritance/Exercise 8 finished/js/script.js

```diff
@@ -0,0 +1,39 @@
+import MyLogger from "Logger";
+import MyEventEmitter from "EventEmitter";
+import MyMovie from "Movies";
+import MyActors from "Actors";
+
+window.onload = function() {
+    const logger = new MyLogger();
+    const EventEmitterObj = new MyEventEmitter();
+    let Matrix1 = new MyMovie("Matrix","1999","183m");
+    let Matrix2 = new MyMovie("Matrix2","2003","168m");  //
+    let Matrix3 = new MyMovie("Matrix3","2005","230m");
+    let Social = {
+        share: function(friendName) {
+            console.log(`${friendName} shares ${Matrix1.title}`);
```

Comment:
Matrix1.title? What happen if I assign this mixin to the object `matrix2`? Do we still want to log matrix 1's title?